Userland/Libraries/LibGfx/BitmapFont.cpp

```diff
@@ -17,7 +17,7 @@ struct [[gnu::packed]] FontFileHeader {
     char magic[4];
     u8 glyph_width;
     u8 glyph_height;
-    u8 type;
+    u16 range_mask_size;
```

Comment:
This change makes `u16 weight` sit kind of awkwardly after 5 bytes, and it's all unaligned. On some platforms (not x86 though), reading it would now cause a SIBGUS.

This would make things line up nicely again:

```
% git diff
diff --git a/Userland/Libraries/LibGfx/BitmapFont.cpp b/Userland/Libraries/LibGfx/BitmapFont.cpp
index d29fc644f6..1c4ea5febf 100644
--- a/Userland/Libraries/LibGfx/BitmapFont.cpp
+++ b/Userland/Libraries/LibGfx/BitmapFont.cpp
@@ -26,8 +26,8 @@ struct [[gnu::packed]] FontFileHeader {
     u8 baseline;
     u8 mean_line;
     u8 presentation_size;
-    u16 weight;
     u8 slope;
+    u16 weight;
     char name[32];
     char family[32];
 };
```

…but that requires regenerating all fonts. Is there a good way for doing font format migrations?